### PR TITLE
Add initial instructions to README

### PR DESCRIPTION
**Pull request by dcodeIO**
This PR adds some initial instructions to the README so interested parties can get an idea what this repo is all about.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,47 @@
...
+Doing so will automatically register common globals like the `Buffer` class and enables requiring
+for example the filesystem module through `import "fs"`.
+
+Documentation
+-------------
+
+There is no sophisticated documentation yet but you can [browse the implementations](./assembly)
+to find out what's already supported (and what's not).
+
+Building
```
Comment (jtenner) [inline README.md:40]: Should make a comment for windows users about

```
npm install -g npm@latest
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,47 @@
...
+As always, if the idea sounds appealing to you, feel free to improve existing APIs or to contribute
+additional ones.
+
+Instructions
+------------
+
+Install the library components as a dependency of your project
+
+```
+$> npm install AssemblyScript/node
```
Comment (jtenner) [inline README.md]: Might like to save them as a `--save-dev` dependency.

**Review by jtenner — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dcodeIO) [reply README.md:40]: Sure, added a few hints to the bottom of that section :)

**Comment by jtenner** [comment]
For documentation we need a road map like AssemblyScript has to keep track of function compatibility

Maybe not examples (yet) or anything fancy, but I like the idea of listing the functions of each node module on different pages (because there's a lot of them)

**Comment by dcodeIO** [comment]
Now links to the wiki with some preliminary links and pages.

**Review by jtenner — APPROVED** [review]

**Review by jtenner — APPROVED** [review]

**Comment by jtenner** [comment]
Do you think this is all set? Or can we add more?

**Comment by dcodeIO** [comment]
Looks like we can continue from here :) Feel free to merge!

**Pull request merged**